Header header
bool motor_enable 
uint8 front_left_pwm_1
uint8 front_left_pwm_2
uint8 front_right_pwm_1
uint8 front_right_pwm_2
uint8 back_left_pwm_1
uint8 back_left_pwm_2
uint8 back_right_pwm_1
uint8 back_right_pwm_2